visualisation/line-graph.ts

```diff
@@ -60,13 +61,18 @@ const xAxis = (): string => {
     "Dec",
   ];
 
-  const ticks = dates.map((date, index) => {
-    const pos = xScale(Date.parse("1 " + date + " 2021"));
+  const many = Array<undefined>(13).fill(undefined); 
+
+  const ticks = many.map((_,index) => {
+    const then = new Date(now);
+    const pos = xScale(then.setUTCMonth(-index));
+    const month = months[then.getUTCMonth()];
+    const year = month === "Jan" ? ' ’' + then.toISOString().slice(2,4) : '';
```

Comment:
might be clearer to do 
```suggestion
    const year = month === "Jan" ? ' ’' + then.getFullYear().toString().slice(-2) : '';
```

Reply:
Added in e3359550e1dc4888368b21629f7e6b70c0ee27a5